homeworks/simonlaszlo/5__oop/car_fleet_mgr.py

```diff
@@ -0,0 +1,101 @@
+
+class Car:
+    """
+    Car osztaly, ajárművek tulajdonságaival.
+    """
+    def __init__(self, brand, model, year):
+        self.brand = brand
+        self.model = model
+        self.year = year
+        self.mileage = 0 
+        self.fuel_level = 100 
+
+    def drive(self, kilometers):
```

Comment:
érdemes a drive-ban is tesztelni hogy a felhasználó nem e ad meg negatív értéket, csak úgy ahogy a refuel()-ben is.

Reply:
A kért változtatást végrehajtottam.